src/routes/(app)/+layout.svelte

```diff
@@ -23,14 +67,12 @@
 <div class="flex w-screen flex-row">
   <Toaster pop={toaster.pop} toasts={toaster.toasts} />
   <div class="sticky top-0 z-20 h-screen w-auto">
-    <Header {user} />
+    <SideNavigation />
   </div>
-  <main
-    id="content"
-    class="min-h-screen w-max flex-auto overflow-auto bg-gray-100"
-  >
+  <main id="content" class="h-screen w-max flex-auto overflow-auto bg-gray-100">
+    <TopNavigation {logout} {namespaceList} />
     <Banners {uiVersionInfo} />
-    <div class="z-10 flex flex-col gap-4 px-10 pb-10 pt-8">
+    <div class="z-10 -mt-4 flex w-full flex-col gap-4 px-10 pb-10 pt-8">
       <ErrorBoundary onError={() => {}}>
         <slot />
       </ErrorBoundary>
```

Comment:
Looks like we've lost the `ScrollToTop` button ⬇️ with the addition of the sticky top nav.

![Screenshot 2023-03-16 at 1 12 30 PM](https://user-images.githubusercontent.com/15069288/225731300-1a4e06c7-5620-42f5-918e-c86a25b89e09.png)

Reply:
Good catch. This one is a little tricker with how I have to do the overflow with sticky for the top-nav but I think i can make it work.